Yoast/yoast-components
# Use replacevar labels (nicenames) in suggestions

## Summary

This PR can be summarized in the following changelog entry:

* Replacevar suggestions now use nicenames rather than raw replacevariable names.

## Test instructions

This PR can be tested by following these steps:

* Test whether the dropdown suggestion list (after typing `%`) uses nicenames, and will fill the nicename into the editor field.
* Verify that the preview still replaces the variable with the correct value.
* Save and load posts, and enter replacevars by just typing `%%title%%` etc., and see that that functionality also still works (so without using the mention/suggestions).

Fixes #577 
Requires https://github.com/Yoast/wordpress-seo/pull/10065 (for custom field replacevars)
